examples/test_all.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I am not sure CI is actually running tests from examples. We should add it to CI config if it does not.